src/types/reviewChanges.ts

```diff
@@ -0,0 +1,92 @@
+export interface ReviewChangesState {
```

Comment:
Maybe you should put all the modules in a separate object?
export interface ReviewChangesState {
    loading: boolean;
    error: string | null;
    **modules: {bank, distribution, gov ...}**
}
So that you can get a separate object with all the keys, for convenient automatic code generation from keys. What do you think?

Reply:
refactored